### ISSUE TITLE
Receiving a DataError at /settings/recovery/codes after setting up MFA using django_mfa 2.0

### ISSUE THREAD

**Issue by mitchdowney** [MicroPyramid/django-mfa#45]
After migrating to django_mfa v2 from v1, and after setting up MFA for a user, I receive the following error message on the recovery code page.

```
Environment:


Request Method: GET
Request URL: http://admin.podverse.fm/settings/recovery/codes/

Django Version: 2.1.5
Python Version: 3.7.4
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'podverse_db.apps.PodverseDbConfig',
 'django_mfa']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'django_mfa.middleware.MfaMiddleware']



Traceback:

File "/usr/local/lib/python3.7/site-packages/django/db/backends/utils.py" in _execute
  85.                 return self.cursor.execute(sql, params)

The above exception (value too long for type character varying(10)
) was the direct cause of the following exception:

File "/usr/local/lib/python3.7/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py" in _get_response
  126.                 response = self.process_exception_by_middleware(e, request)

File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py" in _get_response
  124.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/usr/local/lib/python3.7/site-packages/django/contrib/auth/decorators.py" in _wrapped_view
  21.                 return view_func(request, *args, **kwargs)

File "/usr/local/lib/python3.7/site-packages/django_mfa/views.py" in recovery_codes
  233.                 codes = generate_user_recovery_codes(request.user.id)

File "/usr/local/lib/python3.7/site-packages/django_mfa/views.py" in generate_user_recovery_codes
  220.                 user=UserOTP.objects.get(user=user_id), secret_code=code)

File "/usr/local/lib/python3.7/site-packages/django/db/models/manager.py" in manager_method
  82.                 return getattr(self.get_queryset(), name)(*args, **kwargs)

File "/usr/local/lib/python3.7/site-packages/django/db/models/query.py" in create
  413.         obj.save(force_insert=True, using=self.db)

File "/usr/local/lib/python3.7/site-packages/django/db/models/base.py" in save
  718.                        force_update=force_update, update_fields=update_fields)

File "/usr/local/lib/python3.7/site-packages/django/db/models/base.py" in save_base
  748.             updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)

File "/usr/local/lib/python3.7/site-packages/django/db/models/base.py" in _save_table
  831.             result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)

File "/usr/local/lib/python3.7/site-packages/django/db/models/base.py" in _do_insert
  869.                                using=using, raw=raw)

File "/usr/local/lib/python3.7/site-packages/django/db/models/manager.py" in manager_method
  82.                 return getattr(self.get_queryset(), name)(*args, **kwargs)

File "/usr/local/lib/python3.7/site-packages/django/db/models/query.py" in _insert
  1136.         return query.get_compiler(using=using).execute_sql(return_id)

File "/usr/local/lib/python3.7/site-packages/django/db/models/sql/compiler.py" in execute_sql
  1289.                 cursor.execute(sql, params)

File "/usr/local/lib/python3.7/site-packages/django/db/backends/utils.py" in execute
  100.             return super().execute(sql, params)

File "/usr/local/lib/python3.7/site-packages/django/db/backends/utils.py" in execute
  68.         return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)

File "/usr/local/lib/python3.7/site-packages/django/db/backends/utils.py" in _execute_with_wrappers
  77.         return executor(sql, params, many, context)

File "/usr/local/lib/python3.7/site-packages/django/db/backends/utils.py" in _execute
  85.                 return self.cursor.execute(sql, params)

File "/usr/local/lib/python3.7/site-packages/django/db/utils.py" in __exit__
  89.                 raise dj_exc_value.with_traceback(traceback) from exc_value

File "/usr/local/lib/python3.7/site-packages/django/db/backends/utils.py" in _execute
  85.                 return self.cursor.execute(sql, params)

Exception Type: DataError at /settings/recovery/codes/
Exception Value: value too long for type character varying(10)
```

Could I have done something wrong? Or is there a bug?

**Comment by k0nG** [comment]
Looks like a mismatch between `size_of_recovery_code` here https://github.com/MicroPyramid/django-mfa/blob/master/django_mfa/views.py#L209 and the size specified for the database field here: https://github.com/MicroPyramid/django-mfa/blob/master/django_mfa/models.py#L29

It seems the change was made to the DB field during this commit: https://github.com/MicroPyramid/django-mfa/commit/0eac2ac3e0b64bc26ae1729ff1a41357dbd5e245

It's a simple fix to change, but I'm not sure what the value should be 10 or 16? @chaitu210 are you able to shed some light on why the DB field length was changed?

**Comment by gregkster** [comment]
I noticed the same issue.  Fixed in my Pull Request #46
